File: src/main/java/com/linebot/action/tobuy/TobuyAddInputAction.java
package com.linebot.action.tobuy;

import com.linebot.action.Action;
import com.linecorp.bot.model.message.Message;
import com.linecorp.bot.model.message.TextMessage;
import lombok.AllArgsConstructor;
import lombok.extern.slf4j.Slf4j;
import org.springframework.stereotype.Component;

import javax.annotation.Nullable;
import javax.validation.constraints.NotNull;
import java.util.Arrays;
import java.util.List;

@Slf4j
@AllArgsConstructor
@Component
public class TobuyAddInputAction extends Action {

    @NotNull
    @Override
    public List<Message> execute(@NotNull String userId, @NotNull String message) {
        String messageAdd = "改行区切りで買い物リストを登録してください";
        return Arrays.asList(
                new TextMessage(messageAdd)
        );
    }

    @Nullable
    @Override
    public String getNextAction() {
        return "tobuyAddAction";
    }
}

File: src/main/kotlin/com/linebot/action/tobuy/TobuyAddInputAction.kt
package com.linebot.action.tobuy

import com.linebot.action.Action
import com.linecorp.bot.model.message.Message
import com.linecorp.bot.model.message.TextMessage
import org.springframework.stereotype.Component

@Component
class TobuyAddInputAction : Action() {

    override var nextAction: String? = "tobuyAddAction"

    override fun execute(userId: String, message: String): List<Message> {
        return listOf(TextMessage("改行区切りで買い物リストを登録してください"))
    }
}
